src/app/home/home.component.html

```diff
@@ -62,8 +62,8 @@
   </div>
 </div>
 <!-- Create Space modal -->
-<modal #createSpace title="Create a new collaboration space">
-  <modal-content>
-    <space-dialog></space-dialog>
+<modal #createSpace class="chromeless-modal"  >
+  <modal-content class="chromeless-modal-content" >
+      <space-wizard></space-wizard>
```

Comment:
This dialog still has several more iterations ... can we merge and defer to another pull request